portal2/src/main/java/eu/europeana/portal2/web/presentation/model/abstracts/SearchPageData.java

```diff
@@ -87,6 +87,7 @@
 	private LanguageContainer languageContainer;
 
 	private boolean languagesRemoved;
+    private boolean isNofEnabled = false;
```

Comment:
According to code conventions this would be "nofEnabled", the getter/setter and all other code would stay the same...